src/inputs/search/search/PSearch.vue

```diff
@@ -82,22 +103,125 @@ export default defineComponent<SearchProps>({
             type: Boolean,
             default: undefined,
         },
+        /* context menu fixed style props */
+        visibleMenu: {
+            type: Boolean,
+            default: undefined,
+        },
+        useFixedMenuStyle: {
+            type: Boolean,
+            default: false,
+        },
+        /* context menu props */
+        menu: {
+            type: Array as PropType<MenuItem[]>,
+            default: () => [],
+        },
+        loading: {
+            type: Boolean,
+            default: false,
+        },
+        /* extra props */
+        handler: {
+            type: Function,
+            default: undefined,
+        },
+        disableHandler: {
+            type: Boolean,
+            default: false,
+        },
+        useAutoComplete: {
+            type: Boolean,
+            default: false,
+        },
     },
-    setup(props: SearchProps, { emit, listeners }) {
-        const vm = getCurrentInstance()?.proxy as Vue;
+    setup(props, { emit, listeners }) {
+        const {
+            proxyVisibleMenu, targetRef, targetElement, contextMenuStyle,
+        } = useContextMenuFixedStyle({
+            useFixedMenuStyle: computed(() => props.useFixedMenuStyle),
+            visibleMenu: computed(() => props.visibleMenu),
+        });
+        const contextMenuFixedStyleState = reactive({
+            proxyVisibleMenu, targetRef, targetElement, contextMenuStyle,
+        });
         const state = reactive({
-            proxyIsFocused: makeOptionalProxy('isFocused', vm, false),
+            proxyIsFocused: useProxyValue('isFocused', props, emit),
+            loading: useProxyValue('loading', props, emit),
```

Comment:
I have a question!
The 'loading' state is declared with 'useProxyValue' hook, which means that the 'update:loading' event would be emitted.
1. I wonder why 🤔
2. If there is proper reason, I think 'update:loading' arg type should be declared in `story-helper.ts`.

Reply:
1. Added to manipulate the loading of contenxt-menu when disableHandler is true!
2. thx!!
(Invalid variable name(loading) has been additionally modified!)